es6-destructuring/main.js

```diff
@@ -0,0 +1,50 @@
+const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+
+console.log(book1);
+
+const { title: tbook, author: tauthor, libraryID: bookID } = book1;
+console.log('The title of the book is ' + tbook + ', ' + 'the author is ' + tauthor + ', ' + 'and the library id is ' + bookID);
```

Comment:
@joshuapreciado Make sure `book1`'s property names are destructured into variables with the _same_ name as the properties:

---

![image](https://user-images.githubusercontent.com/16143771/212814295-cb2b3bb2-be0c-430c-8b40-0ef44f54c7b2.png)